### PR TITLE
Update install.md

### PR DESCRIPTION
**Pull request by kevinhammond**
Fixed description so that it can be applied to a vanilla installation

### CODE REVIEW

File: doc/getting-started/install.md [hunk 1]
```diff
@@ -107,27 +129,44 @@ Change the working directory to the downloaded source code folder:
...
+    git fetch --all --recurse-submodules --tags
     git tag
     git checkout tags/<TAGGED VERSION>
 
+#### Configure the build options
+
+We turn off compiler optimisations, and explicitly use the GHC version that we installed earlier.  This avoids
+possible problems with prevously installed versions of GHC.
+
+    cabal configure -O0 -w ghc-8.10.2
```
Comment (dcoutts) [inline doc/getting-started/install.md:141]:
```suggestion
We explicitly use the GHC version that we installed earlier.  This avoids
defaulting to a system version of GHC that might be older.

    cabal configure -w ghc-8.10.2
```

**Review by dcoutts — APPROVED** [review]
LGTM with the exception of the -O0 thing below.

**Review by intricate — APPROVED** [review]

**Comment by kevinhammond** [comment]
Do I just keep waiting for bors or commit again and hope it goes through this time?

**Comment by kevinhammond** [comment]
Thanks!  This is a bit frustrating :)

**Comment by intricate** [comment]
> Thanks! This is a bit frustrating :)

That it is :smiley:

**Pull request merged**